taxi/views.py

```diff
@@ -28,6 +32,21 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required()
+def car_driver_add(request, pk):
+    print(request.__dict__)
```

Comment:
The `print(request.__dict__)` statement is likely used for debugging purposes and should be removed in production code to avoid unnecessary console output and potential exposure of sensitive information.